README.md

```diff
@@ -1,68 +1,70 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<a href="https://github.com/Mapptastic/"><img src="https://avatars2.githubusercontent.com/u/58233055?s=200&v=4" title="Mapptastic" alt="Mapptastic"></a>
 
-## Available Scripts
+# UTM Finder
 
-In the project directory, you can run:
+> A simple React app to find UTM zone.
 
-### `yarn start`
+- [![Build Status](http://img.shields.io/travis/badges/badgerbadgerbadger.svg?style=flat-square)](https://travis-ci.org/badges/badgerbadgerbadger) 
+- [![Coverage Status](http://img.shields.io/coveralls/badges/badgerbadgerbadger.svg?style=flat-square)](https://coveralls.io/r/badges/badgerbadgerbadger)
+- [![License](http://img.shields.io/:license-mit-blue.svg?style=flat-square)](http://badges.mit-license.org) 
```

Comment:
This icons is not related with out project please remove it, we may add something like this for circle ci
`<img alt="CircleCI" src="https://img.shields.io/circleci/build/github/Mapptastic/utmfinder/master">`